AUDIO/LockRadioStationTrackList.md

```diff
@@ -1,16 +1,13 @@
 ---
 ns: AUDIO
 ---
-## _LOCK_RADIO_STATION_TRACK_LIST
+## LOCK_RADIO_STATION_TRACK_LIST
 
 ```c
 // 0xFF5E5EA2DCEEACF3
-void _LOCK_RADIO_STATION_TRACK_LIST(char* radioStation, char* trackListName);
+void LOCK_RADIO_STATION_TRACK_LIST(char* radioStation, char* trackListName);
 ```
 
```

Comment:
If you want to add a description - `Locks the specified radio station track list`

Reply:
Adding a description here is redundant, the name describes what it does.